### ISSUE TITLE
[a11y] Cannot use QCBX search to select new Agent in Security panel

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1289]
**Describe the bug**
Cannot use QCBX search to select new Agent in Security panel

**To Reproduce**
1. Go to Security Panel
2. Try to reassign the agent using the search function on the QCBX
3. See error

**Screenshots**
![image](https://user-images.githubusercontent.com/37256050/164298326-ecb6a57f-2169-4f33-bf74-1d14bab6359a.png)

**Desktop (please complete the following information):**
macOS, Chrome

**Comment by grantfitzsimmons** [comment]
![msedge_zHHpdLmQwQ](https://user-images.githubusercontent.com/37256050/166007578-39d1680c-652f-4c16-b037-1577cc3efd7c.gif)

Cannot use keyboard to select agent OR the mouse to select an agent. Could be related- if not, I can open a bug.

This is in the CE form on `freshfish`
